pydantic/_internal/_fields.py

```diff
@@ -110,6 +111,8 @@ def collect_model_fields(  # noqa: C901
             continue
         if ann_name.startswith('_'):
             continue
+        if cls.__pydantic_root_model__ and ann_name != 'root':
```

Comment:
I'd like to create a constant with the root field name or shall we put it in the model config maybe?

Reply:
🤷 I don't think there's much risk of confusing the root field name. Please don't put it in the model config, we should try to reduce/eliminate that as much as possible imo. We could theoretically add if people need it for some reason but I think it's okay for now to just require the use of `root` as the name.